code/src/main/webapp/CSS/style.css

```diff
@@ -1,124 +1,307 @@
 body, html {
+  color: #222;
   height: 100%;
   margin: 0;
-  color: #222;
 }
 
 h1, p, a {
+  color:#025696;
+  font-family: Muli;
+  text-shadow: 0 0 18px rgba(255, 255, 255, 1);
   margin: 0;
 }
 
 div {
   position: fixed;
-  width: 100%;
   text-align: center;
+  width: 100%;
 }
 
-#wrapper-background {
-  height:100%;
+.fade-in {
+  animation: fadeIn ease 1s;
+  -webkit-animation: fadeIn ease 1s;
+  -moz-animation: fadeIn ease 1s;
+  -o-animation: fadeIn ease 1s;
+  -ms-animation: fadeIn ease 1s;
+}
+.fade-out {
+  animation: fadeOut ease 1s;
+  -webkit-animation: fadeOut ease 1s;
+  -moz-animation: fadeOut ease 1s;
+  -o-animation: fadeOut ease 1s;
+  -ms-animation: fadeOut ease 1s;
+}
+
+.FAQ {
```

Comment:
meganit - css classes should alwasy be lower-dash-case

The reasons for this are actually pretty messed up - older internet browsers don't always support upper cased CSS Properties!

Reply:
using faq instead of FAQ throughout will be good!